spatial-tracking-explainer.md

```diff
@@ -149,6 +144,8 @@ function onSessionStarted(session) {
 }
 ```
 
+It is important to note that `XRViewerPose` objects retrieved using `local` and `local-floor` reference spaces may include position information as well as rotation information.  For example, hardware which does not support 6DOF tracking (ex: GearVR) may still use neck-modeling to improve user comfort. Similarly, a user may lean side-to-side on a device with 6DOF tracking (ex: HTC Vive). The result is that `local` and `local-floor` experiences must be resilient to position changes despite not being dependent on receiving them.
```

Comment:
Maybe this should go into the section above? I'm not sure how to make sure this info doesn't get lost in the sea of words ^_^;

Reply:
Moved it up a section and partially repeated that paragraph again down here. It does feel less likely to get overlooked that way.